proposals/propertyDependencies-adr.md

```diff
@@ -1,14 +1,16 @@
 # Add New Keyword: `propertyDependencies`
 
-* Status: proposed
-* Deciders: @gregsdennis, @jdesrosiers, @relequestual
-* Date: 2022-04-07
+- Status: proposed
+- Deciders: @gregsdennis, @jdesrosiers, @relequestual
+- Date: 2022-04-07
 
 Technical Story:
 
-- Issue discussing feature - https://github.com/json-schema-org/json-schema-spec/issues/1082
-- PR to add to the spec - https://github.com/json-schema-org/json-schema-spec/pull/1143
-- ADR to extract from the spec and use feature life cycle - https://github.com/json-schema-org/json-schema-spec/pull/1505
+- Issue discussing feature - <https://github.com/json-schema-org/json-schema-spec/issues/1082>
```

Comment:
Why the angle brackets?

Reply:
The angle brackets are markdown syntax for URLs. We aren't used to using them because Github-flavored markdown detects URLs automatically and turns them into links. The markdown style guide the linting plugin is based on [prefers the explicit syntax](https://cirosantilli.com/markdown-style-guide/#automatic-links-without-angle-brackets). I think the rationale makes sense. By forcing you to use angle brackets or backticks, it helps catch errors where a URL is converted to a link when it wasn't intended to be.